src/utils/helpers.ts

```diff
@@ -18,3 +19,15 @@ export const comparePurses = (a: PursesJSONState, b: PursesJSONState) =>
 
 export const sortPurses = (purses: PursesJSONState[]) =>
   purses.sort(comparePurses);
+
+export const mapAtom = <T1, T2>() => {
```

Comment:
helpful to distinguish what T1 and T2 are, keys and values
```suggestion
export const mapAtom = <K, V>() => {
```